src/api.js

```diff
@@ -4,8 +4,8 @@ const ncNewsApi = axios.create({
   baseURL: "https://nc-news-z5u7.onrender.com/api",
 });
 
-export function getArticles(p) {
-  return ncNewsApi.get(`/articles?p=${p}`).then((response) => {
+export function getArticles(p, topic = "") {
+  return ncNewsApi.get(`/articles?p=${p}&topic=${topic}`).then((response) => {
```

Comment:
We could potentially refactor this to use Axios Params, where the topic is just an optional parameter that is passed in. 👍